### PR TITLE
Add fuzzymatcher

### PR DESCRIPTION
**Pull request by Maerville**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by Maerville** [comment]
@conda-forge/help-python :)

File: recipes/fuzzymatcher/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  sha256: 24fd3bb3c347523802af378e711aef48200759bc498546599a0652f39f3e9242
+
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
```
Comment (marcelotrevisani) [inline recipes/fuzzymatcher/meta.yaml:20]:
```suggestion
```

File: recipes/fuzzymatcher/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  run:
+    - python >=3.5
+    - pandas >=0.20.3
+    - metaphone
+    - fuzzywuzzy
+    - python-dateutil
+
+test:
+  imports:
+    - fuzzymatcher
```
Comment (marcelotrevisani) [inline recipes/fuzzymatcher/meta.yaml:33]: can we try to add ``pip check`` here please?
```suggestion
    - fuzzymatcher
  requires:
    - pip
  commands:
    - pip check
```

File: recipes/fuzzymatcher/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
+{% set name = "fuzzymatcher" %}
+{% set version = "0.0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+
```
Comment (marcelotrevisani) [inline recipes/fuzzymatcher/meta.yaml:10]:
```suggestion
```

**Comment by Maerville** [comment]
@conda-forge/help-python
@conda-forge-admin

**Comment by Maerville** [comment]
@conda-forge/help-python
@conda-forge-admin

:(

**Review by dopplershift — APPROVED** [review]

**Pull request merged**